src/BenchmarksDriver/README.md

```diff
@@ -21,15 +21,14 @@ Options:
   --kestrelThreadCount   Maps to KestrelServerOptions.ThreadCount.
   -n|--scenario          Benchmark scenario to run
   -m|--scheme            Scheme (http or https).  Default is http.
-  -o|--source            Source dependency. Format is 'repo@branchOrCommit'. Repo can be a full URL, or a short name under https://github.com/aspnet.
   -w|--webHost           WebHost (e.g., KestrelLibuv, KestrelSockets, HttpSys). Default is KestrelSockets.
   --aspnetCoreVersion    ASP.NET Core packages version (Current, Latest, or custom value). Current is the latest public version (2.0.*), Latest is the currently developped one. Default is Latest (2.1-*).
   --runtimeVersion       .NET Core Runtime version (Current, Latest, Edge or custom value). Current is the latest public version, Latest is the one enlisted, Edge is the latest available. Default is Latest (2.1.0-*).
   --arguments            Arguments to pass to the application. (e.g., "--raw true")
   --port                 The port used to request the benchmarked application. Default is 5000.
   -r|--repository        Git repository containing the project to test.
   --projectFile          Relative path of the project to test in the repository. (e.g., "src/Benchmarks/Benchmarks.csproj)"
-  --useRuntimeStore      Runs the benchmarks using the runtime store if available.
+  --runtime-store        Runs the benchmarks using the runtime store (2.0) or shared aspnet framework (2.1).
```

Comment:
ZOMG breaking changes. Make sure you publish another global tool.

Reply:
I will. But nobody is using this parameter right now, because it's broken (remember the hard-coded netcoreapp2.0 string)